bindings/rust/lib.rs

```diff
@@ -97,7 +97,7 @@ impl<'a> MarkdownCursor<'a> {
     /// current node is an inline or block node.
     ///
     /// See also [`field_name`](Self::field_name).
-    pub fn field_id(&self) -> Option<u16> {
```

Comment:
I guess this is not really forward compatible (which would mean a major version bump) but I don't think it should cause big trouble for anyone.

Reply:
The type was changed upstream in `0.21.0`.